core/kamon-core/src/main/scala/kamon/trace/Trace.scala

```diff
@@ -56,6 +58,17 @@ trait Trace {
     */
   def keep(): Unit
 
+  /**
+    * Signals that a Span belonging to this trace has failed.
+    */
+  def spanFailed(): Unit
+
+  /**
+    * Returns the number of failed Spans for this trace in this process. This error count does not reflect errors that
+    * might have happened on other services participating in the same trace.
+    */
+  def failedSpanCount(): Int
```

Comment:
WDYT, Ivan? 🤔 
```suggestion
  def failedSpansCount(): Int
```

Reply:
Thanks @ihostage, got it done and pushed!